src/Http/ApiClient.php

```diff
@@ -21,6 +21,15 @@
  */
 class ApiClient
 {
+    /**
+     * Error constants.
+     */
+    const ERROR_ENVIRONMENT_TYPE_UNKNOWN = 'Unknown environmentType "%s"';
+    const ERROR_MAC_OS_CURL_VERSION = 'Your PHP seems to be linked to the MacOS provided curl binary. ' .
+    'This is incompatible with our SDK, please reinstall by running: "brew reinstall %s --with-homebrew-curl".%s';
+    const ERROR_CURL_DOES_NOT_SUPPORT_ROOT_CA_PINNING =
+        'Curl does not support root CA pinning. See https://curl.haxx.se/docs/todo.html#Support_intermediate_root_pinn';
```

Comment:
Why did you change the entire sentence..... The formulation is incorrect now; curl does not support root "Certificate Authority" pinning. 😂 The Authorities are not pinned, it's the certificate that gets pinned.

Reply:
🤦‍♂️